### ISSUE TITLE
Standard installation through pip and conda

### ISSUE THREAD

**Issue by cbouy** [chemosim-lab/ProLIF#6]
**TL;DR: don't `pip install prolif` yet**

ProLIF currently depends on a [custom fork](https://github.com/cbouy/mdanalysis/tree/prolif) of MDAnalysis which is not available on the PyPI repositories. This is because ProLIF depends on features that are not yet merged in the main branch of MDAnalysis, and that will be available in one of their next release. 

Until all the needed features are released on PyPI, I have to rely on this custom fork, which makes it impossible to upload ProLIF v0.3.0 and above to PyPI (see this [build](https://github.com/chemosim-lab/ProLIF/runs/1597752573?check_suite_focus=true) for more technical info).

The package that is currently installed with `pip install prolif` is a much older version that corresponds to [this](https://github.com/cbouy/ProLIF) GitHub repository that I decided to deprecate as it has bugs, is untested, and could only read some MOL2 files.

Once the MDAnalysis update is on PyPI, I'll upload the releases there. For `conda`, I'm guessing the issue will be the same (I doubt that I can have dependencies that aren't on conda/pip) so I'll wait for this as well.

For now, stay safe and `pip install git+https://github.com/chemosim-lab/ProLIF.git`, or even better, refer to the [installation instructions](https://prolif.readthedocs.io/en/latest/source/installation.html).

**Comment by lordmiss** [comment]
Hi @cbouy,

Thank you for your detailed explanation and great work. I understood.

Stay healthy!
Hanjo

**Comment by hadim** [comment]
For the record here is the conda package: https://github.com/conda-forge/staged-recipes/pull/17499 which is currently blocked by mdanalysis (require the 2.1.0 to be released first).

**Comment by IAlibay** [comment]
I'm likely to release MDA 2.1.0 over the coming week or so. If this needs to be in 2.1.0 may I ask for someone to push for https://github.com/MDAnalysis/mdanalysis/pull/3044 to finish review? (I've completely lost track of where that PR stands, if it's close to completion then let's aim to get it merged before 2.1.0).

**Comment by cbouy** [comment]
@IAlibay sure, from what I remember it was close to being done, maybe missing a few tests.
I've just rebased the branch from develop, I'll do my best to tackle test failures and missing coverage when that's done.

**Comment by cbouy** [comment]
Quick update to avoid any confusion: it's now scheduled for MDAnalysis **v2.2.0**.

This way the MDAnalysis core developers can release 2.1.0 very soon without having to wait for me, and 2.2.0 will come out shortly after (technically as soon as I'm done working on the corresponding PR).

**Comment by hadim** [comment]
Thanks for the update @cbouy.

Just ping again once ready, and I will warp up the PR.

**Comment by cbouy** [comment]
ProLIF 1.0.0 is now available on both pip and conda-forge!
